app/scripts/TilesetFinder.js

```diff
@@ -200,70 +188,142 @@ class TilesetFinder extends React.Component {
   }
 
   handleSearchChange() {
-    const domElement = ReactDOM.findDOMNode(this.searchBox);
+    const domElement = this.searchBox;
 
     this.setState({ filter: domElement.value });
   }
 
+  /*
+   * Create the nested checkbox tree by partitioning the
+   * list of available datasets according to their group
+   *
+   * @param {dict} datasetsDict A dictionary of id -> tileset
+   *  def mappings
+   * @param {string} filter A string to filter the results by
+   * @returns {list} A list of items that define the nested checkbox
```

Comment:
I'd use proper JS dtypes: `dict` => `object` and `list` => `array`

Reply:
Sure.